deploy/deploy.rb

```diff
@@ -23,14 +23,7 @@
 append :linked_dirs, "bundle", "log", "tmp/pids", "tmp/cache", "tmp/sockets", "public/system"
 
 # Configure capistrano-bundler
-set :bundle_roles, :all                                         # this is default
-set :bundle_servers, -> { release_roles(fetch(:bundle_roles)) } # this is default
-set :bundle_path, -> { shared_path.join("bundle") }             # this is default
-set :bundle_without, (["development", "test"] - [ENV["RAILS_ENV"]]).join(" ")
-set :bundle_flags, "--deployment"
-set :bundle_env_variables, {}                                   # this is default
-set :bundle_clean_options, ""                                   # this is default
-set :bundle_jobs, 4                                             # default: nil
+set :bundle_roles, :none
```

Comment:
I'm not sure if this is the best way to disable capistrano's bundler stuff, but it gets the job done.